src/ClientApp/src/Page.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { BrowserRouter, Route } from 'react-router-dom';
+import { CandleChartPageHoc } from './components/charts/CandleChartPageHoc';
+import { Home } from './components/main/Home';
+import { Layout } from './components/main/Layout';
+import { ContentService } from './services/ContentService';
+import { AllContent, Language } from './services/Models';
+import { HtmlUtils } from './utils/HtmlUtils';
+
+export interface PageState {
+    allContent: AllContent;
+}
+
+export class Page extends React.Component<{}, PageState> {
+
+    private contentService: ContentService;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Private member variable 'contentService' is never reassigned; mark it as 'readonly'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS2933)